frameworks/Java/gemini/gemini-mysql.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM resin:latest
```

Comment:
I'm reading in the [Docker docs](https://docs.docker.com/engine/reference/builder/#from) that

> Each `FROM` instruction clears any state created by previous instructions.

Does that mean `FROM` is basically single inheritance?  If yes, then how would I declare multiple dependencies?

Reply:
Yes, and you cannot; instead, you chain the inheritance. I have not yet seen a need for multiple inheritance, but if we have one we can probably just tackle it via having some redundant installs in docker images (not great, but not terrible).